src/components/Modals/Send/hooks/useSendDetails/useSendDetails.tsx

```diff
@@ -52,6 +52,7 @@ const moduleLogger = logger.child({
 export const useSendDetails = (): UseSendDetailsReturnType => {
   const [fieldName, setFieldName] = useState<AmountFieldName>(SendFormFields.CryptoAmount)
   const [loading, setLoading] = useState<boolean>(false)
+  const [curInput, setCurInput] = useState<string>('')
```

Comment:
```suggestion
  const [currentInput, setCurrentInput] = useState<string>('')
```
Use full terminology, we generally don't use abbreviated forms except for indexes

Reply:
Getting reverted.